src/MailingListBundle/Block/Service/ContactsManager.php

```diff
@@ -0,0 +1,113 @@
+<?php
+
+namespace Opifer\MailingListBundle\Block\Service;
+
+use Symfony\Component\DependencyInjection\ContainerInterface as Container;
+use Doctrine\ORM\EntityManagerInterface;
+use Symfony\Component\HttpKernel\Log\LoggerInterface;
+use Opifer\MailingListBundle\Entity\MailingList;
+use GuzzleHttp\Client as GuzzleClient;
+use GuzzleHttp\HandlerStack;
+use GuzzleHttp\Subscriber\Oauth\Oauth1;
+
+class ContactsManager
+{
+	/** @var string */
+    protected $consumer_key;
+
+    /** @var string */
+    protected $consumer_secret;
+
+    /** @var Service Container */
+    protected $service_container;
+
+    /** @var LoggerInterface */
+    protected $logger;
+
+    public function __construct(EntityManagerInterface $em, Container $container, LoggerInterface $logger, $consumer_key, $consumer_secret)
+    {
+        $this->em = $em;
+        $this->consumer_key = $consumer_key;
+        $this->consumer_secret = $consumer_secret;
+        $this->container = $container;
+        $this->logger = $logger;
+    }
+
+    public function addContactToMailPlus(MailingList $contact)
```

Comment:
Please add docblocks.

And a Mailinglist is not the same as a contact / subscription. So please change the typehinting here.